src/main/resources/application.properties

```diff
@@ -3,6 +3,52 @@ DOMAIN=localdomain
 OAUTH_CLIENT_ID=null
 OAUTH_SECRET=null
 
+SKIP_SSL_VALIDATION="true"
+SPRING_PROFILES_ACTIVE=cloud,pki-geoserver-auth
```

Comment:
This needs to be removed from the application.properties file. The profiles are to be configured via the environment in the JenkinsFile. Otherwise we have issues when adding/removing them, since you can't really concatenate/remove from the list.

Reply:
Done.